build-go.sh

```diff
@@ -144,7 +145,7 @@ function doBuild() {
 
 	mkdir -p "$dir"
 
-	if ! (cd "$build_dir_name" && goBuild "$package" "$goos" "$goarch") > build-log; then
+	if ! (cd "$build_dir_name" && goBuild "$package" "$goos" "$goarch") > "$build_dir_name/build-log" 2>&1; then
```

Comment:
:information_source:   why error logs were not created correctly OR were missing stderr